ui-particles-angular/projects/ui-particles-angular/src/lib/oem-theme/gio-top-bar/gio-top-bar-content/gio-top-bar-content.component.scss

```diff
@@ -37,12 +38,21 @@ $typography: map.get(gio.$mat-theme, typography);
   }
 
   &.apim {
-    background: rgb(0 111 185 / 50%);
-    color: map.get(gio.$mat-cyan-palette, default-contrast);
+    background: oem.$oem-top-bar-apim-button-background;
+    color: map.get(gio.$mat-oem-palette, background-contrast);
   }
 
   &.cockpit {
     background: rgb(135 111 236 / 40%);
     color: map.get(gio.$mat-accent-palette, default-contrast);
   }
 }
+
+::ng-deep {
```

Comment:
what about use :host-context(gio-top-bar-content) into gio-top-bar-menu ? 

in the world of angular components. it's up to the child to adapt to its environment, not the opposite :p